notebooks/ch14/bias_modeling.md

```diff
@@ -81,6 +81,43 @@ The second term in this expression, $ \text{Var}(f_\hat{\theta}(z)) $, represent
 
 The third and final term in this expression, $ \text{Var}(\epsilon) $, represents the irreducible error or the noise in the data generation and collection process. This term is small when the data generation and collection process is precise or has low variation. This term is large when the data contain large amounts of noise.
 
+We start with the mean squared error
+
+$$\mathbb{E}[(\gamma - f_{\hat{\theta}}(z))^2]$$
+
+and square the inside term and apply linearity of expectation:
+
+$$ =\mathbb{E}[\gamma^2 -2\gamma f_{\hat{\theta}} +f_\hat{\theta}(z)^2]$$  
+$$= \mathbb{E}[\gamma^2] - \mathbb{E}[2\gamma f_{\hat{\theta}}(z)] + \mathbb{E}[f_{\hat{\theta}}(z)^2]$$
+
+We then substitute $h(z) + \epsilon$ for $\gamma$:
+
+$$ =\mathbb{E}[(h(z) + \epsilon)^2] - \mathbb{E}[2(h(z) + \epsilon)] \mathbb{E}[f_{\hat{\theta}}(z)] + \mathbb{E}[f_{\hat{\theta}}(z)^2]$$          
+
+Simplifiying some more: (Note that $\mathbb{E}[h(z)] = h(z)$ because $h(z)$ is a deterministic function.)   
+
+
+$$ =\mathbb{E}[h(z)^2 -2h(z) \epsilon + \epsilon^2] - (2h(z) + \mathbb{E}[2\epsilon]) \mathbb{E}[f_{\hat{\theta}}(z)] + \mathbb{E}[f_{\hat{\theta}}(z)^2]$$ 
+
+$$= h(z)^2 + 2h(z)\mathbb{E}[\epsilon] + \mathbb{E}[\epsilon^2] - (2h(z) + 2\mathbb{E}[\epsilon]) \mathbb{E}[f_{\hat{\theta}}(z)] + \mathbb{E}[f_{\hat{\theta}}(z)^2]$$   
```

Comment:
I believe the first + here should be a - instead. Also maybe add something along the lines of "applying the linearity of expectation again" before this